content/en/docs/ops/configuration/traffic-management/dns-proxy/index.md

```diff
@@ -7,10 +7,6 @@ owner: istio/wg-networking-maintainers
 test: no
 ---
 
-{{< warning >}}
-This feature is currently [experimental](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE.md) and considered `alpha`.
-{{< /warning >}}
-
```

Comment:
Removed per this discussion: https://github.com/istio/istio.io/pull/8772#discussion_r557495467